lib/vrl/tests/tests/internal/closure_arg_kind.vrl

```diff
@@ -0,0 +1,38 @@
+# result: [
+#           {
+#             "log_events": {
+#               "id": "35683658089614582423604394983260738922885519999578275840",
+#               "message": "{\"bytes\":26780,\"datetime\":\"14/Sep/2020:11:45:41 -0400\",\"host\":\"157.130.216.193\",\"method\":\"PUT\",\"protocol\":\"HTTP/1.0\",\"referer\":\"https://www.principalcross-platform.io/markets/ubiquitous\",\"request\":\"/expedite/convergence\",\"source_type\":\"stdin\",\"status\":301,\"user-identifier\":\"-\"}",
+#               "timestamp": "2020-09-14T19:09:29.039Z"
+#             },
```

Comment:
```suggestion
#             "id": "35683658089614582423604394983260738922885519999578275840",
#             "message": "{\"bytes\":26780,\"datetime\":\"14/Sep/2020:11:45:41 -0400\",\"host\":\"157.130.216.193\",\"method\":\"PUT\",\"protocol\":\"HTTP/1.0\",\"referer\":\"https://www.principalcross-platform.io/markets/ubiquitous\",\"request\":\"/expedite/convergence\",\"source_type\":\"stdin\",\"status\":301,\"user-identifier\":\"-\"}",
#             "timestamp": "2020-09-14T19:09:29.039Z",
```

Post `map_values` the `.log_events` object should be removed and the contents at the root.

Reply:
oh, yea. I needed to assign the result of `map_values` back to `.`